<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectorListClass</class>
 <widget class="QWidget" name="DetectorListClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模型列表</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pb_load">
       <property name="text">
        <string>加载模型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_unload">
       <property name="text">
        <string>卸载模型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_start">
       <property name="text">
        <string>开始检测</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_Add">
       <property name="text">
        <string>添加模型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_Edit">
       <property name="text">
        <string>编辑模型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_Del">
       <property name="text">
        <string>删除模型</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QTreeWidget" name="tw_Detectors">
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
